### DISCUSSION TITLE
ETF Saving Plan

### DISCUSSION THREAD

**Discussion by svenboeer** [ghostfolio/ghostfolio#1647] [Ideas]
Hey Guys,

would it be possible to store your ETF savings plan under the activities so that this does not have to be entered constantly Manuel.

Helpful here would also be a selection whether this is saved several times a month, for example, at the beginning and middle of the month or only once a month.

of course the order should be entered as activity, but automatically :)

best regards

**Comment by dtslvr** [comment]
Thank you for bringing your idea to the table.

How would you determine the unit price for the recurring transaction to match with the broker?

**Reply by rtrive** [reply]
I think Ghostfolio could use a default broker and then if one wants to change it, it could do it manually. What do you think?

**Reply by rodlcopter13** [reply]
How about to adapt this from Portfolio Performance?
There you can add transaction fees for every Saving Plan.
![grafik](https://user-images.githubusercontent.com/125887414/234269407-4a56c3c3-98e7-4f9d-955b-3aa13345981c.png)


For calculations I would suggest to do this just like in Portfolio Performance:
(Saving Rate - Transaction Fee)/Unit Price = Quantity
![grafik](https://user-images.githubusercontent.com/125887414/234269457-64775719-1ad0-47f2-b0b5-df4c89303af8.png)

**Comment by matzingerhh** [comment]
Hi, i was working also on this topic. In the end i ended up with Excel  and generated a CSV activity Import. I track some saving plans and manual creating would be a lot of work.
If we would use it like proposed the real pieces and unit price would never fit. But if the system would create the activity in general an the data have to be reworked manually from the broker order details this would at least save some time. So a kind of preliminary activity generated from master data as proposed could be something helpful.
